pkg/aws/rds.go

```diff
@@ -0,0 +1,91 @@
+
+package aws
+
+import (
+	"database/sql"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+
+func GetWhetherDatabaseExistsInRdsPostgresInstance(t *testing.T, dbUrl string, dbPort int32, dbUsername string, dbPassword string, databaseName string) bool {
+	output, err := GetWhetherDatabaseExistsInRdsPostgresInstanceE(t, dbUrl, dbPort, dbUsername, dbPassword, databaseName)
+	require.NoError(t, err)
+	return output
+}
+
+func GetWhetherDatabaseExistsInRdsPostgresInstanceE(t *testing.T, dbUrl string, dbPort int32, dbUsername string, dbPassword string, databaseName string) (bool, error) {
+	connectionString := fmt.Sprintf("host=%s port=%d user=%s password=%s dbname=%s", dbUrl, dbPort, dbUsername, dbPassword, databaseName)
+
+	db, connErr := sql.Open("pgx", connectionString)
+	if connErr != nil {
+		return false, connErr
+	}
+	defer db.Close()
+	return true, nil
+}
+
+func GetWhetherSchemaExistsInRdsPostgresInstance(t *testing.T, dbUrl string, dbPort int32, dbUsername string, dbPassword string, databaseName string, expectedSchemaName string) bool {
```

Comment:
Consider renaming this to `SchemaExistsInRdsPostgresInstance`, the `GetWhether` pattern is long and awkward.